### PR TITLE
fix: await realtime chat context updates

### PR DESCRIPTION
**Pull request by mykmelez**
## Summary

- Await `AgentActivity.updateChatCtx()` from `Agent.updateChatCtx()` when an activity is active.
- Await `RealtimeSession.updateChatCtx()` from `AgentActivity.updateChatCtx()` when a realtime session is active.
- Add focused tests covering both async boundaries.
- Add a patch changeset for `@livekit/agents`.

## Why

`Agent.updateChatCtx()` is public async API, but the active realtime path could return before the provider-side chat context sync completed. That allowed an immediately following model turn to race conversation item create/delete acknowledgements and observe stale context.

The realtime session contract already exposes `updateChatCtx()` as a `Promise<void>` and OpenAI Realtime waits for server acknowledgements internally. This change propagates that promise through the voice agent layers so callers can reliably sequence follow-up turns after context sync completes.

This also matches the Python SDK shape, where `Agent.update_chat_ctx()` and `AgentActivity.update_chat_ctx()` await the realtime session update.

## Validation

- `pnpm exec vitest run agents/src/voice/agent.test.ts`
- `pnpm exec vitest run plugins/openai/src/realtime/realtime_model.test.ts`
- `pnpm exec eslint agents/src/voice/agent.ts agents/src/voice/agent_activity.ts agents/src/voice/agent.test.ts` (warnings only, pre-existing `no-explicit-any` warnings)
- `pnpm --filter @livekit/agents build`
- `pnpm --filter @livekit/agents-plugins-test build`
- `pnpm --filter @livekit/agents-plugin-silero build`
- `pnpm --filter @livekit/agents-plugin-openai build`
- Real OpenAI Realtime smoke: update chat context, immediately ask a follow-up turn, verify the model sees the updated context.

### CODE REVIEW

File: .changeset/await-realtime-chatctx-sync.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@livekit/agents': patch
+---
+
+Await active realtime chat context updates through `Agent.updateChatCtx()` so callers can reliably sequence follow-up model turns after conversation item sync completes.
```
[the remark this answers is not in the document]
Comment (mykmelez) [reply .changeset/await-realtime-chatctx-sync.md:5]: Thanks for checking. I think this file should stay as-is because `.changeset/**` is covered by the repo-level REUSE annotations rather than inline SPDX headers. `REUSE.toml` explicitly annotates `.changeset/**` under “trivial files” with LiveKit copyright and Apache-2.0, and existing changesets on `main` such as `.changeset/assemblyai-inference-model.md` also omit inline SPDX headers. So this appears consistent with the repo convention.
